### ISSUE TITLE
A little questions about the training.

### ISSUE THREAD

**Issue by mxmxlwlw** [davidsandberg/facenet#410]
I used mtcnn to get the aligned face of WebFace. And I used the data to train center loss using the program in "https://github.com/ydwen/caffe-face". The  problem is that it seems the network can't converge without overfitting. My training data consist of 368775 imgs and my testing data consist of 122767 imgs. Is it because of less of data? Or I just didn't train my network well. Please help me!

**Comment by mxmxlwlw** [comment]
Can you help me out? It seems that the data of WebFace is not enough if doing rectify.

**Comment by davidsandberg** [comment]
With less data you would probably have to use a little bit more weight decay. But I haven't looked into caffe-face that much.
